pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -573,8 +573,7 @@ func applyHTTPRouteDestination(
 	} else {
 		action.ClusterSpecifier = &route.RouteAction_WeightedClusters{
 			WeightedClusters: &route.WeightedCluster{
-				Clusters:    weighted,
-				TotalWeight: wrappers.UInt32(totalWeight),
```

Comment:
remove the totalWeight var entirely maybe?

Reply:
good point. removed